common/services/prismic/transformers/collection-venues.ts

```diff
@@ -92,3 +92,17 @@ export function transformCollectionVenues(
     return Number(a.order) - Number(b.order);
   });
 }
+
+// venue is passed down as JSON, so need to convert the date strings back to Moment objects
+export function convertTimeStringsBackToMoments(venue: Venue): Venue {
```

Comment:
Minor: while we're here, maybe rename this to match the event-based helper? `fixVenueDatesInJson`?

It's more consistent and we won't have to rename this function if/when we ditch Moment.